library/unwind/src/lib.rs

```diff
@@ -1,10 +1,12 @@
 #![no_std]
 #![unstable(feature = "panic_unwind", issue = "32837")]
 #![feature(link_cfg)]
+#![feature(native_link_modifiers)]
+#![feature(native_link_modifiers_bundle)]
 #![feature(nll)]
 #![feature(staged_api)]
-#![feature(unwind_attributes)]
 #![feature(static_nobundle)]
```

Comment:
Nit: this feature should no longer be necessary now.

Reply:
libunwind.rs still need it:
```
error[E0658]: the `#[unwind]` attribute is an experimental feature
  --> library/unwind/src/libunwind.rs:92:5
   |
92 |     #[unwind(allowed)]
   |     ^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #58760 <https://github.com/rust-lang/rust/issues/58760> for more information
   = help: add `#![feature(unwind_attributes)]` to the crate attributes to enable

error[E0658]: the `#[unwind]` attribute is an experimental feature
   --> library/unwind/src/libunwind.rs:244:9
    |
244 |         #[unwind(allowed)]
    |         ^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #58760 <https://github.com/rust-lang/rust/issues/58760> for more information
    = help: add `#![feature(unwind_attributes)]` to the crate attributes to enable

error: aborting due to 2 previous errors
```

Reply:
I was talking about `feature(static_nobundle)`.

Reply:
It's still needed in current compiler:
```
error[E0658]: kind="static-nobundle" is unstable
  --> library/unwind/src/libunwind.rs:84:5
   |
84 | /     link(
85 | |         name = "unwind",
86 | |         kind = "static",
87 | |         modifiers = "-bundle",
88 | |         cfg(not(target_feature = "crt-static"))
89 | |     )
   | |_____^
   |
   = note: see issue #37403 <https://github.com/rust-lang/rust/issues/37403> for more information
   = help: add `#![feature(static_nobundle)]` to the crate attributes to enable

error[E0658]: kind="static-nobundle" is unstable
   --> library/unwind/src/libunwind.rs:117:9
    |
117 |         link(name = "unwind", kind = "static", modifiers = "-bundle", cfg(not(target_feature = "crt-static")))
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #37403 <https://github.com/rust-lang/rust/issues/37403> for more information
    = help: add `#![feature(static_nobundle)]` to the crate attributes to enable

error[E0658]: kind="static-nobundle" is unstable
   --> library/unwind/src/libunwind.rs:241:9
    |
241 |         link(name = "unwind", kind = "static", modifiers = "-bundle", cfg(not(target_feature = "crt-static")))
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #37403 <https://github.com/rust-lang/rust/issues/37403> for more information
    = help: add `#![feature(static_nobundle)]` to the crate attributes to enable

error[E0658]: kind="static-nobundle" is unstable
  --> library/unwind/src/lib.rs:46:9
   |
46 |         #[link(name = "unwind", kind = "static", modifiers = "-bundle")]
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #37403 <https://github.com/rust-lang/rust/issues/37403> for more information
   = help: add `#![feature(static_nobundle)]` to the crate attributes to enable

```

This feature gate is checked in https://github.com/rust-lang/rust/blob/master/compiler/rustc_metadata/src/native_libs.rs#L255 while link modifier is checked in https://github.com/rust-lang/rust/blob/master/compiler/rustc_ast_passes/src/feature_gate.rs#L330

Reply:
Ah, I see, this is somewhat unexpected.